torcharrow/functional.py

```diff
@@ -141,6 +141,60 @@ def add_tokens(
     return _dispatch("add_tokens", input_col, tokens, begin)
 
 
+def bpe_tokenize(
+    tokenizer,
+    input_col: Union[ListColumn, List[str]],
+) -> ListColumn:
+    """
+    Tokenize text into a list of tokens using the GPT-2 BPE Tokenizer
+
+    Parameters
+    ----------
+    tokenizer: An instance of the `GPT2BPEEncoder` class
+    input_col: List of input text
+
+    Examples
+    --------
+    >>> import torcharrow as ta
+    >>> from torcharrow import functional
+    >>> # init_bpe_encoder() is helper function that creates an instance of the `GPT2BPEEncoder` class
+    >>> tokenizer = init_bpe_encoder()
+    >>> a = ta.column([["Hello World!, how are you?", "Respublica superiorem"]])
+    >>> functional.bpe_tokenize(tokenizer, a)
+    0  ['15496', '2159', '28265', '703', '389', '345', '30']
+    1  ['4965', '11377', '64', '2208', '72', '29625']
+    dtype: List(String(nullable=True), nullable=True), length: 2, null_count: 0
+    """
+    return _dispatch("bpe_tokenize", tokenizer, input_col)
```

Comment:
can check`_ta.is_built_with_torch()` and raise a more explicit error message? (otherwise, it will just say "`bpe_tokenize` function not found~)